docs/CreateAzureADforAzureBot.md

```diff
@@ -2,19 +2,19 @@
 
 If you would like to authenticate against your own Azure AD for development or private use of the AzureBot code, you'll need to setup your own Azure AD application.  Here are the steps you need to follow:
 
-1. Create or Utilize an Azure Active Directory (AAD) Tenant
-2. Create a new Azure AD app on the AAD tenant
-3. Configure the AAD application for use with the AzureBot
-4. Configure App Multi-Tenancy (optional)
+- [Create or Utilize an Azure Active Directory (AAD) Tenant](#createtenant)
```

Comment:
Ok, I'll reverse the numbering for AAD. I did start with pre-requisites for the dev one, but then most of it turned into a pre-requisite section other than the 'putting it all together' part so ended up leaving it all as part of the higher level.